### ISSUE TITLE
FeliCa sniffing issues

### ISSUE THREAD

**Issue by doegox** [RfidResearchGroup/proxmark3#444]
Let's discuss felica sniffing here to not pollute #442 too much.

* Sniffing phone <> RC-S966 (Lite-S): ok, but tag responses > 16 are truncated
* Sniffing proxmark3 <> RC-S966 (Lite-S): no result at all
* Sniffing phone <> RC-S886 / RC-S965: only reader communication is visible. Only on one attempt, the tag communication was visible (and the reader commands were not visible!)

**Comment by iceman1001** [comment]
Yup.  How about @7homasSutter get the reader / raw command finished and stable before jumping on the sniff part.

**Comment by iceman1001** [comment]
With latest fixes,  we might even be able to close this one :)

**Comment by doegox** [comment]
I've to test the new one
